pygmt/datasets/earth_relief.py

```diff
@@ -14,9 +17,9 @@
 @kwargs_to_strings(region="sequence")
 def load_earth_relief(
     resolution="01d",
-    region=None,
-    registration=None,
-    data_source="igpp",
+    region: Union[str, Sequence, None] = None,
+    registration: Literal["gridline", "pixel", None] = None,
+    data_source: Literal["igpp", "gebco", "gebcosi", "synbath"] = "igpp",
```

Comment:
It is probably not possible to get:
- double quotation marks (`" "`) instead of single quotation marks (`' '`)
- an upper-case `Default`

![283153676-d904cd92-67f1-4306-a73a-775a79229b4a_cut](https://github.com/GenericMappingTools/pygmt/assets/94163266/479cf620-9c08-4020-b36b-1f19e3bedae0)

Reply:
I don't think it's possible.